disputer/disputer.js

```diff
@@ -71,37 +73,52 @@ class Disputer {
           at: "Disputer",
           message: "Cannot dispute liquidation: not enough collateral (or large enough approval) to initiate dispute.",
           id: disputeableLiquidation.id,
-          sponsor: disputeableLiquidation.sponsor
-        });
-
-        Logger.debug({
-          at: "Disputer",
-          message: "Dispute call error message",
-          id: disputeableLiquidation.id,
           sponsor: disputeableLiquidation.sponsor,
           error: error
         });
         continue;
       }
 
-      // TODO: handle transaction failures.
-      const receipt = await dispute.send({
-        from: this.account,
-        gas: 1500000,
-        gasPrice: this.gasEstimator.getCurrentFastPrice()
-      });
+      disputePromises.push(
+        dispute
+          .send({
+            from: this.account,
+            gas: 1500000,
+            gasPrice: this.gasEstimator.getCurrentFastPrice()
+          })
+          .catch(error => {
+            Logger.error({
+              at: "Disputer",
+              message: `Failed to dispute liquidation: ${error.message}`,
```

Comment:
I print out the error message here, since I have yet to see the actual "error" property ever getting filled out on an error